main/swadge2024.c

```diff
@@ -349,6 +350,11 @@ void app_main(void)
             }
         }
 
+        if (ESP_NOW == cSwadgeMode->wifiMode)
```

Comment:
This was slightly different last year: https://github.com/AEFeinstein/Super-2023-Swadge-FW/blob/6755896c2380330c8fe55c2e4a232bc0760d1dd1/main/swadge_esp32.c#L615C1-L615C1

Reply:
Oh, it calls that even in IMMEDIATE mode? I guess that's fine then, I can get rid of the emulator hack part then.